scrape/apps.go

```diff
@@ -115,6 +115,8 @@ type AppManifest struct {
 	Name *string `json:"name,omitempty"`
 	//Required. The homepage of your GitHub App.
 	URL *string `json:"url,omitempty"`
+	// The full URL of the endpoint to authenticate users via the GitHub App.
+	CallbackURL *string `json:"url,omitempty"`
```

Comment:
This change makes no sense to me, based on line 117 above it. 
Note that both fields have identical JSON bindings, which means they get mapped to the same thing:

```
	URL *string `json:"url,omitempty"`
	CallbackURL *string `json:"url,omitempty"`
```